app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -390,6 +396,12 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event) {
     if (event.target === inputRef.current) {
+      if (forceNativeCamera) {
+        addPageAction(`IdV: Force native camera. Failed attempts: ${failedCaptureAttempts}`, {
+          field: name,
+        });
```

Comment:
Since `addPageAction` logs to the event.log, the name passed should be constant and the failed attempts should go in the data hash that will be logged with it.

```suggestion
        addPageAction('IdV: Force native camera.', {
          field: name,
          failed_attempts: $(failedCaptureAttempts),
        });
```

Reply:
Should we also allowlist this event? see [EVENT_MAP](https://github.com/18F/identity-idp/blob/3a4017bb65ea69a2b65bb0e6420d1e0de7a69d54/app/controllers/frontend_log_controller.rb#L8) for methods on [AnalyticsEvents](https://github.com/18F/identity-idp/blob/3a4017bb65ea69a2b65bb0e6420d1e0de7a69d54/app/services/analytics_events.rb)
1. so it doesn't need to get prefixed with "frontend"
2. so we can get documentation for it?

Reply:
I think we should probably leave out the period as well in the suggested change, for consistency with other events?

Reply:
Going to work on this now. What are the arguments passed to these ruby-side logging methods in AnalyticsEvents -- are they just the same as the dict/object we pass as the second arg to `addPageAction` on the frontend?

Reply:
> Going to work on this now. What are the arguments passed to these ruby-side logging methods in AnalyticsEvents -- are they just the same as the dict/object we pass as the second arg to `addPageAction` on the frontend?

Yeah, the properties we add in the JavaScript object here will be passed through as keyword arguments to the corresponding Ruby method.

https://github.com/18F/identity-idp/blob/e174eb1c1bb46447a5fa9cd2ab0fd0a097820c34/app/controllers/frontend_log_controller.rb#L34

Reply:
Ok I believe I've dealt with this in https://github.com/18F/identity-idp/pull/6727/commits/1b78fed6582520bc3959ebb97ab89126e6de97c7 and subsequent commits